### PR TITLE
keepass2: add page

### PR DESCRIPTION
**Pull request by sbrl**
Another command! There's more that you could do with it, but this is a good starting point.

### CODE REVIEW

File: pages/common/keepass2.md [hunk 1]
```diff
@@ -0,0 +1,15 @@
+# keepass2
+
+> A free, open source, light-weight and easy-to-use password manager.
```
Comment (waldyrious) [inline pages/common/keepass2.md]: FOSS is arguably irrelevant (most of the commands we cover are); "light-weight" is acceptable; but "easy-to-use" is too subjective and opinionated. I'd suggest going with a plain, objective description -- `A light-weight password manager` would work.

**Comment by waldyrious** [comment]
LGTM apart from the description change I suggested inline.

**Comment by sbrl** [comment]
Ah, right. I just lifted that description from the KeePass website

**Comment by sbrl** [comment]
There - is that better?

File: pages/common/keepass2.md [hunk 2]
```diff
@@ -0,0 +1,15 @@
...
+
+`keepass2`
+
+- Start KeePass 2, opening a specific password database:
+
+`keepass2 "{{/path/to/database.kbdx}}"`
+
+- Use a specific key file to open a password database:
+
+`keepass2 "{{/path/to/database.kbdx}}" -keyfile:{{/path/to/key/file.key}}`
```
Comment (agnivade) [inline pages/common/keepass2.md]: Shouldn't there be quotes around the key file path too ?

And how is the key file generated for a particular db ?

Comment (waldyrious) [inline pages/common/keepass2.md]: Or neither. We don't usually quote paths. Is it particularly necessary in this case?

By the way, do the paths need to be absolute? If not, maybe remove the leading slash.

Comment (sbrl) [inline pages/common/keepass2.md]: > Shouldn't there be quotes around the key file path too ?

Nope

> And how is the key file generated for a particular db ?

It's created when you first set up a database,  I think. I don't use it personally.

> Or neither. We don't usually quote paths. Is it particularly necessary in this case?

Nope! This is just my personal preference.

> By the way, do the paths need to be absolute? If not, maybe remove the leading slash.

No again!

**Pull request merged**